test/Generator/PropertyGeneratorTest.php

```diff
@@ -330,4 +380,20 @@ public function testFromReflectionOmitsTypeHintInTypedProperty(): void
 
         self::assertSame('    private $typedProperty;', $code);
     }
+
+    public function testFromReflectionReadonlyProperty(): void
+    {
+        if (false === strpos((string)phpversion(), '8.1')) {
+            $this->markTestSkipped('This test can only run on PHP ^8.1');
+
+            return;
+        }
+
+        $reflectionProperty = new PropertyReflection(ClassWithReadonlyProperty::class, 'readonly');
+
+        $generator = PropertyGenerator::fromReflection($reflectionProperty);
+        $code      = $generator->generate();
+
+        self::assertSame('    private $typedProperty;', $code);
```

Comment:
This assertion seems to indicate that the `readonly` flag was not added? Seems wrong